CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 Records breaking changes from major version bumps
 
+## 33.0.0
+
+Reverting our usage of an interim alphagov fork of odfpy now that our patch has been merged into master and released.
+1
+ACTION: update your `requirements-app.txt`, removing the alphagov odfpy
+github URI.
```

Comment:
Might like to add "...if it exists", because apps that don't care about ods probably never added it and this may leave some people head-scratching.